pom.xml

```diff
@@ -172,22 +173,27 @@
         <dependency>
             <groupId>io.github.openfeign</groupId>
             <artifactId>feign-okhttp</artifactId>
-            <version>9.3.1</version>
+            <version>${feign.version}</version>
         </dependency>
         <dependency>
             <groupId>io.github.openfeign</groupId>
             <artifactId>feign-jackson</artifactId>
-            <version>9.3.1</version>
+            <version>${feign.version}</version>
         </dependency>
         <dependency>
             <groupId>io.github.openfeign</groupId>
             <artifactId>feign-slf4j</artifactId>
-            <version>9.3.1</version>
+            <version>${feign.version}</version>
         </dependency>
         <dependency>
             <groupId>io.github.openfeign.form</groupId>
             <artifactId>feign-form</artifactId>
-            <version>2.2.1</version>
+            <version>3.8.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.openfeign.form</groupId>
+            <artifactId>feign-form-spring</artifactId>
```

Comment:
Where is it used?

Reply:
There is not direct usages, i my local environment to make develop branch working I do not use it.
So If there is not some indirect impact - I would suggest to remove it too

Reply:
@acumarav Could you check if login remains working w/o `feign-form-spring`?
If it wouldn't be broken then we could eliminate `feign-form-spring` as unnecessary.

Reply:
@wivern - I am able to login without this dependency, but I see some error at token refresh in both cases with and without that dependency

Reply:
I believe token refresh is not about feign. Let's remove redundant dependency.

Reply:
@acumarav thank you for test, and also could you check import of Atlas entities?